src/pages/Researchers/ProgramPostForm.js

```diff
@@ -75,19 +75,20 @@ const ProgramPostForm = () => {
       console.log(values);
       console.log(selectedTags);
 
-      await Db.addProgram(userId, values, selectedTags).then;
+      await Db.addProgram(userId, values, selectedTags);
+
+      setSent(true);
+      notification.open({
+        message: "Success!",
+        description: "New post created.",
+      });
+      navigate(`/programs`);
     } catch (error) {
       notification.open({
         message: "Error!",
         description: error.message,
       });
     }
-    setSent(true);
-    notification.open({
-      message: "Success!",
-      description: "New post created.",
-    });
-    navigate(`/programs`);
```

Comment:
Moved them into the try block so that the success notification and navigation does not happen even when there's an error in Db.addProgram. My bad, forgot to look through carefully previously